mmengine/hooks/sync_buffer_hook.py

```diff
@@ -13,6 +13,18 @@ class SyncBuffersHook(Hook):
 
     def __init__(self) -> None:
         self.distributed = is_distributed()
+        self.called_in_train = False
+
+    def before_val_epoch(self, runner) -> None:
+        """All-reduce model buffers before each validation epoch.
+
+        Args:
+            runner (Runner): The runner of the training process.
+        """
```

Comment:
Suggest adding a short description to explain why we need to implement `before_val_epoch` method.

Reply:
I have added such description, please check on that. Thanks!